resources/todomvc/big-dom-generator/utils/buildComplex.js

```diff
@@ -56,6 +56,13 @@ function buildComplex(CALLER_DIRECTORY, SOURCE_DIRECTORY, TITLE, FILES_TO_MOVE)
         head.appendChild(cssLink);
     }
 
+    for (const cssFile of EXTRA_CSS_TO_LINK) {
+        const extraCssLink = doc.createElement("link");
+        extraCssLink.rel = "stylesheet";
+        extraCssLink.href = cssFile;
+        head.appendChild(extraCssLink);
+    }
```

Comment:
I would say we just move `CSS_FILES_TO_ADD_LINKS_FOR` outside of this function and then define it at each call site.

or at least combine the two for loops.

```suggestion
    for (const cssFile of [...CSS_FILES_TO_ADD_LINKS_FOR, ...EXTRA_CSS_TO_LINK]) {
        const cssLink = doc.createElement("link");
        cssLink.rel = "stylesheet";
        cssLink.href = cssFile;
        head.appendChild(cssLink);
    }
```
WDYT

Reply:
Done